sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
 # Release History
 
-## 1.0.0-beta.7 (Unreleased)
+## 1.0.1 (Unreleased)
 
```

Comment:
Add breaking changes introduced in this change, e.g. ChatAsyncClient ctor change

Reply:
Also add "New Features" or "Added" for real time notification